packages/ui-components/stories/UITable.story.tsx

```diff
@@ -182,4 +182,4 @@ export const EditableTable2 = (): JSX.Element => {
             </div>
         </Stack>
     );
-};
\ No newline at end of file
+};
```

Comment:
Did the linter add this file? Don't think this change should be part of this PR

Reply:
I only ran linting inside axios-extension. It might be pre-commit hook triggered this. Reverted this out-of-scope change from the PR.